5-Cookies/client-react/src/components/TodoList/TodoListItem/TodoListItem.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import { mockTodo } from "../../../test/utils";
+import { TodoListItemDriver } from "./TodoListItem.driver";
+
+describe("todo list item component", () => {
+  let driver : TodoListItemDriver;
+
+  beforeEach(() => {
+    driver = new TodoListItemDriver();
+  });
+
+  test("should display description text", async () => {
+    const item = { text: "test" };
+    driver.given.item(item);
```

Comment:
return the driver itself and use chaining when possible
`driver.given.item(item).when.render()`

Reply:
done